planning/development-strategy.md

```diff
@@ -42,6 +43,57 @@
    -  [ ] uppercase letter
    -  [ ] placed on the right side
 
+
+## `for: body` (_label_)
+
+### Our Program section
+### `type: html` (_label_)
+   -  [ ] Title (_issue_)
+      
```

Comment:
You mentioned the HTML elements in the CSS part but you didn't describe them here.